spec/plans/remote.fmf

```diff
@@ -0,0 +1,23 @@
+summary: Referencing remote plans
+story:
+    As a tester I want to reference remote plans
+    in order to reuse the config without duplication.
+description:
```

Comment:
Can remote contain list of dictionaries?
Name has to match exactly one plan, right? Is it regexp (as --name options usually are)?

Reply:
Only one plan can be used and no matching is applied. The spec updated to be explicit about this.